cinema_service/settings.py

```diff
@@ -137,3 +137,11 @@
 # https://docs.djangoproject.com/en/4.0/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
+
+MEDIA_ROOT = BASE_DIR / "media"
+
+MEDIA_URL = "/media/"
+
+REST_FRAMEWORK = {
+
+}
```

Comment:
The `REST_FRAMEWORK` dictionary is empty. Please configure it with appropriate settings such as authentication classes and pagination to ensure the API functions correctly.